packages/remix-dev/config.ts

```diff
@@ -422,9 +422,8 @@ function addTrailingSlash(path: string): string {
   return path.endsWith("/") ? path : path + "/";
 }
 
-const entryExts = [".js", ".jsx", ".ts", ".tsx"];
-
 function findEntry(dir: string, basename: string): string | undefined {
+  const entryExts = [".js", ".jsx", ".ts", ".tsx"];
```

Comment:
This like an improvement as `extryExt` wasn't used anywhere else

Reply:
I disagree.

Although this const is only used in the `findEntry`, it will be re-created on each method call.  
Having it outside of the method call only creates the const once, which is a cleaner way to do imo.

Reply:
> it will be re-created on each method call.
True, i haven't dug into this area of the code & don't know how often this gets called. If this gets called only once during dev or build time, then the `const` being defined inside just makes sure its get's garbage collected more quickly, immediately after the function is called

I do see your point though

Reply:
I definitely didn't notice the fact that the function was called in more places (currently 4 calls). Probably the "change" doesn't impact as much in terms of performance, but it makes more sense to keep it that way, since we don't know if the number of calls will increase or not.